src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -184,7 +185,7 @@ export const validateAddress = (
   payload,
   analyticsSectionName,
 ) => async dispatch => {
-  const addressPayload = { address: { ...payload } };
+  const addressPayload = { address: payload };
```

Comment:
Shouldn't we keep this as a destructuring so we don't unintentionally mutate payload?

Reply:
@Clark-W-Griswold this should be fine as its not used outside the scope of this function and regardless of where its used in the function, it requires the changes to payload that I am making.  Either way, I am not overly opinionated on this and can change it back if you'd like.

Reply:
Probably best not to mutate :)